config/default.yml

```diff
@@ -538,6 +538,7 @@ Style/HashSyntax:
     - ruby19
     - ruby19_no_mixed_keys
     - hash_rockets
+    - no_mixed_keys
```

Comment:
I think that the supported styles could use some clarity. I don't think that it is very obvious what the difference between some of the styles is. It could be improved with some documentation of what each style does along with some examples in the cop itself.